envs/scIB-R-integration.yml

```diff
@@ -284,7 +287,7 @@ dependencies:
     - pytest==5.3.1
     - pytz==2019.3
     - rpy2==3.0.5
-    - scanpy==1.4.5.dev114+gd69832a
+    - scanpy==1.4.5
```

Comment:
does this downgrade affect anything important in our pipeline? Maybe better to use a git install, no?

Reply:
not sure where this version comes from and if there was any  new feature that we needed @danielStrobl Do you remember how we installed it and why we didn't use the pip version?
I'm guessing that we might have had some computational issues, but `scanpy==1.4.5` has been working well so far. It's also the same version that we're using in the python environment

Reply:
If anything, I think we should have same versions for the python and R environments, i.e. 1.4.6

Reply:
I think there was one bug in the Scanpy code that we wanted to avoid, so we fixed it to this version. Unfortunately the slack messages for this are long gone.

Reply:
1.4.6 sounds good to me. Could you adapt and test @mumichae ?

Reply:
Updating to 1.4.6 didn't change the metrics. Most likely scanpy isn't used in this environment at all.